### ISSUE TITLE
Chimera to Uncover

### ISSUE THREAD

**Issue by Ybenbetka** [pwn20wndstuff/Undecimus#1966]
Hello, Is it possible to switch from chimera to uncover wihtout loss my sources and tweaks ?
(I have cydia installed with my chimera jailbreak).

Thank you.

**Comment by SarKaa** [comment]
Yes it should be, but you will lose Sileo, which shouldn't be a problem

**Comment by Ybenbetka** [comment]
> Oui, ça devrait l'être, mais vous perdrez Sileo, ce qui ne devrait pas être un problème

**Comment by Ybenbetka** [comment]
if i want to come back to chimera later, will i find my repo and tweaks on sileo?

**Comment by rlatm** [comment]
> if i want to come back to chimera later, will i find my repo and tweaks on sileo?

If you are concerned, you may want to do a backup with https://github.com/captinc/Batchomatic
Please close this issue if now resolved. Thank you.
